### ISSUE TITLE
3 full stars seems intuitively wrong when level not completed (so before even attempted)

### ISSUE THREAD

**Issue by AndersBTC** [JWWeatherman/math_bot#328]
In this case then even though I have not attempted level 2 yet on Conditionals, then it shows three full stars. As we know it is always the case that three full stars are shown, even when the level is not attempted, but I think it is intuitively wrong. When I see that I feel I have completed it, which is not the case. So I am suggesting a third option apart from the stars and the lock. It could be something as simple as a "Try now" text or logo, but really I am not the UI or designer and don't know what would be best. I am just requesting the feature. 

Although I can see there is a problem with previously passed levels passed with videos, because they should also signal to try again, which they do nicely (as long as they don't go automatically three full stars after one hour). So I guess this is a UI challenge, but maybe not one worth attacking.. just wanted to give my input.

![screenshot 2018-09-10 18 27 25](https://user-images.githubusercontent.com/39171116/45384947-5a6b5080-b5c4-11e8-8f7e-683422af98ea.png)

**Comment by JWWeatherman** [comment]
I agree. This is a good bug, but we need a good designer to give this some thought. We have a lot going on with stars and it might be better to just break it into two controls if we can't find a clever way to address all of the issues.

**Comment by AndersBTC** [comment]
Totally agree!
